### PR TITLE
Update CHI 2025

### PR DESCRIPTION
**Pull request by sci-m-wang**
### Which conference does this PR update?

- CHI 2025

### Related URL

- https://chi2025.acm.org/

### CODE REVIEW

**Review by PAN-Ziyue — CHANGES_REQUESTED** [review]

File: conference/HI/chi.yml [hunk 1]
```diff
@@ -13,3 +13,19 @@
       timezone: AoE
       date: May 11 - May 16, 2024
       place: Honolulu, Hawaii, USA
+
+- title: CHI
+  description: ACM Conference on Human Factors in Computing Systems
+  sub: HI
+  rank: A
+  dblp: chi
+  confs:
```
Comment (PAN-Ziyue) [inline conference/HI/chi.yml:22]: Duplicate parts?

**Review by PAN-Ziyue — CHANGES_REQUESTED** [review]

**Comment by jacklightChen** [comment]
The CHI 2025 has been updated by [#527], plz consider removing the related parts, thx.

**Comment by sci-m-wang** [comment]
Ok, I just note it. It may not have been merged yet when I viewed the page.

**Comment by jacklightChen** [comment]
> Ok, I just note it. It may not have been merged yet when I viewed the page.

Yes, that's quite a coincidence, but you can still make a pr for updating the other records.

**Pull request closed without merging**